### ISSUE TITLE
bw cli 2024.4.0 appears broken (again)

### ISSUE THREAD

**Issue by luckman212** [blacs30/bitwarden-alfred-workflow#195]
reminds me of #186 

I just filed an issue...

https://github.com/bitwarden/clients/issues/9142

Is everyone else having problems too?

**Comment by rpop0** [comment]
Yeah it's broken, I can't use the workflow for a couple of months.  Seems like the project is abandoned.

**Comment by luckman212** [comment]
Sad to say but I believe that yes it's time to throw in the towel on Bitwarden CLI. They just don't seem to care about it, and it's been broken in various ways for months. Even when it is working, it's too slow (for me) to be very useful. And then you start getting into leaving cached versions of your vault lying around on the disk which probably isn't a great idea security-wise.

I switched to 1Password last week and honestly I can't believe how much better it is, literally in EVERY way. Only wish I had done it sooner, as I've lost days of my life to debugging and working around silly Bitwarden issues.

**Comment by blacs30** [comment]
I am testing currently this workflow, which uses the bitwarden cli only but has all important features and more.
https://github.com/ajrosen/Bitwarden-Accelerator

1Password is of course very nice and more powerful, specially their global autofill.
